mowgli_etl/pipeline/wdc/wdc_product_dimensions.py

```diff
@@ -1,22 +1,27 @@
-from typing import NamedTuple, Optional
+from dataclasses import dataclass
+from typing import Optional
 
+from dataclasses_json import dataclass_json
 
-class WdcProductDimensions(NamedTuple):
+
+@dataclass_json
+@dataclass(frozen=True)
+class WdcProductDimensions:
     """
     Should we handle volume, mass dimensions?
     How do we handle multiple dimensions?
     How do we handle clothing dimensions?
     """
 
-    depth: Optional[float]
-    height: Optional[float]
-    length: Optional[float]
-    width: Optional[float]
-    volume: Optional[float]
-    mass: Optional[float]
-    depth_unit: Optional[str]
-    height_unit: Optional[str]
-    length_unit: Optional[str]
-    width_unit: Optional[str]
-    volume_unit: Optional[str]
-    mass_unit: Optional[str]
+    depth: Optional[float] = None
```

Comment:
Please go back and split these as we discussed, @dataclass WdcProductDimension ( value, unit ).